packages/lodestar/src/sync/index.ts

```diff
@@ -81,7 +81,7 @@ export class Sync extends EventEmitter {
   };
 
   public async start(): Promise<void> {
-    //await new Promise((resolve) => this.network.once("peer:connect", resolve));
+    await new Promise((resolve) => this.network.once("peer:connect", resolve));
```

Comment:
This is not ok fix, thats why we commented it out. If we don't have peers, we are stuck here.
You should implement more complex system like, pause syncing if peers drop to zero and continue if we get new peer.

There is also question if we should queue blocks when we don't have peers

Reply:
Totally. I do not plan on having that there in the end 😂 
I just pushed this so that someone who's working on something and only needs syncing to "kinda work" can use this code to test or something :)